components/Map/index.js

```diff
@@ -395,13 +400,13 @@ class MapContainer extends React.Component {
           initialState={this.initialState}
         />
         <CookieNotice />
-        {(isDbLoading || isMapLoading) ? (
+        {(isDbLoading || isMapLoading) && acknowledgeModalShown === false ? (
           <>
             <LoadingModal />
             <FunFactCard />
           </>
         ) : (
-          <AcknowledgeModal/>
+          <AcknowledgeModal onClose={this.onClose}/>
```

Comment:
the condition `acknowledgeModalShown` should be place here at line 408 to conditionally render the `AcknowledgeModal`.

```
        ) : !acknowledgeModalShown ? (
          <AcknowledgeModal onClose={this.onClose}/>
        ) : null}
```